frontend/src/scenes/PreflightCheck/preflightLogic.tsx

```diff
@@ -214,7 +214,10 @@ export const preflightLogic = kea<preflightLogicType>([
         siteUrlMisconfigured: [
             (s) => [s.preflight],
             (preflight): boolean => {
-                return Boolean(preflight && (!preflight.site_url || preflight.site_url != window.location.origin))
+                if (window.location.port === '6006') {
+                    return false // Disable this warning in Storybook
```

Comment:
Not sure which warning that is, but this won't work for https://storybook.posthog.net/

Reply:
Good catch, updated this "fix" to work in that case too, and explained the reason